test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -271,6 +271,89 @@ function ExecuteRequestWithCustomUserAgent {
     return $result
 }
 
+function GetSelfSignedCert {
```

Comment:
I did [here](https://github.com/PowerShell/PowerShell/pull/4546#issuecomment-321892193) and with [305c048](https://github.com/PowerShell/PowerShell/pull/4546/commits/305c048399d7a2119c63a717eb05bb312fe78a97). Was that insufficient?